installing/installing_oci/installing-oci-assisted-installer.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oci-assisted-installer"]
+= Using the Assisted Installer to install a cluster on OCI
+include::_attributes/common-attributes.adoc[]
+:context: installing-oci-assisted-installer
+
+toc::[]
+
+In {product-title} {product-version}, you can use the Assisted Installer to install a cluster on {oci-first} by using infrastructure that you provide.
+
+// The Assisted Installer and OCI overview
+include::modules/installing-oci-about-assisted-installer.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../installing/installing_platform_agnostic/installing-platform-agnostic.html#cluster-entitlements_installing-platform-agnostic[Internet access for {product-title}]
+* link:https://docs.oracle.com/en-us/iaas/Content/Block/Concepts/blockvolumeperformance.htm#vpus[Volume Performance Units]
+* xref:../../installing/installing_on_prem_assisted/installing-on-prem-assisted.html#using-the-assisted-installer_installing-on-prem-assisted[Using the Assisted Installer]
```

Comment:
This xref is not going to help the user install with AI. Better to link to the splash page of Assisted Installer for OCP. Don't link directly to the guide because you would have to maintain the link for new vesions.

Reply:
HI @apinnick . Sorry what do you mean by SPlash Page? Are you discussing the customer portal here?

Reply:
She means this page 

https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14 

though a direct link may be preferable if possible.